README.md

```diff
@@ -141,6 +141,24 @@ Run the binary using `ego` so that it can be run in the secure enclave.
 AZDCAP_DEBUG_LOG_LEVEL=INFO ego run doracled gen-oracle-key
 ```
 
+### verify remote report
+
+You can verify that key is generated in SGX using the promised binary.
+For that, the public key and its remote report are required.
+
+```json
+{
+  "public_key_base_64" : "<base64-encoded-public-key>",
+  "remote_report_base_64": "<base64-encoded-remote-report>"
+}
+```
+
+Then, you can verify the remote report.
+
+```bash
+AZDCAP_DEBUG_LOG_LEVEL=INFO ego run doracled verify-report <remote-report-path>
```

Comment:
Could you explain what's the reason of having this CLI? Is this for someone who wants to verify SGX reports uploaded to the Panacea if they want (for example, by a genesis oracle)?

Reply:
Exactly. Using this CLI, anyone can verify the report from genesis oracle.

Reply:
If so, it would be more suitable to modify the JSON file as below:
```json
{
  "remote_report_base64": "...",
  "remote_report_data_base64: "..."
}
```
, because remote reports can contain any data such as a public key or so.

Reply:
Ah. I think it’s not a big deal. Let’s keep going as you wish.